test/geometry/data/simple-cms.gdml

```diff
@@ -4,6 +4,23 @@
   <define/>
 
   <materials>
+    <isotope N="1" Z="1" name="H1">
```

Comment:
Do we want to keep these files consistent when they appear in multiple places (e.g. app/demo-loop/simple-cms.gdml), or maybe just keep one version, or does it not matter?

Reply:
I was mulling over the same question. I think since the `app/demo-loop` and `test/geometry/data` directories are so far apart, the improved locality is worth the annoyance of duplication. It also means we can continue to improve or change the demo loop's test to make it more realistic while keeping this test case as-is.